Data-tranformations/fairgenomes/experiment.yarrrml.yaml

```diff
@@ -5,18 +5,36 @@ prefixes:
   EDAM: https://bioportal.bioontology.org/ontologies/EDAM?p=classes&conceptid=
   FIX: http://purl.obolibrary.org/obo/FIX_
   GENEPIO: http://purl.obolibrary.org/obo/GENEPIO_
+  fgs: https://w3id.org/fair-genomes/ontology/ 
 
 
-mappings:
+mappings: # mapping for sequencing
   fairgenomes:
     sources:
       - [experiment.csv~csv] # I am going to name it like this 
-    s: ega.dataset:$(accession_id)
+    s: ega.runs:$(accession_id) # I guess that experiment is included in ega.runs otherwise I cannot think something more appropriate from the list
     po:
-      - [a, EDAM:topic_3168] # I based at experiment is regarded as sequencing
-      - [OMIABIS:0000006, ega.dataset:$(accession_id)~iri] # I am unsure for this one, I will only keep it 
-      - [NCIT:C171337, $(accession_id)] #thi is new, based on sequencing identifier
-      - [FIX:0000704, $(design_description)] #this is new
-      - [GENEPIO:0000071, $(platform)] #this is new
-      - [GENEPIO:0001921, $(model)] #this is new
-      - [OMIABIS:0000006, $(study_accession_id)] #this is new 
\ No newline at end of file
+      - [a, EDAM:topic_3168] # "Sequencing"
+      - [a, fgs:Sequencing]
+      - [OMIABIS:0000006, ega.runs:$(accession_id)~iri] # I am unsure for this one, I changed the ega.datasets to ega runs but this OMIABIS:0000006 is not correct. I recommend another identifier in the next line
+      - [NCIT:C171337, $(accession_id)] # "Sequencing Assay Unique Identifier"
+      - [fgs:Sequencing_Sequencing_identifier, $(description)]
+      - [FIX:0000704, $(design_description)] # "Sequencing method"
+      - [fgs:Sequencing_Sequencing_method, $(description)]
+      - [GENEPIO:0000071, $(platform)] # "Sequencing platform"
+      - [fgs:Sequencing_Sequencing_platform, $(description)]
+      - [GENEPIO:0001921, $(model)] # "Sequencing instrument model"
+      - [fgs:Sequencing_Sequencing_instrument_model, $(description)]
+
+
+mappings: # mapping for study 
```

Comment:
I wrote in a previous message, do we have to write all these again `sources` and `s` when we are doing the mapping of the other parts such as study here ?